hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -713,7 +741,14 @@
       name: chip_aes_idle
       desc: '''Verify AES idle signaling to clkmgr.
 
-            Details TBD.
+            - Write the AES clk hint to 1 within clkmgr to indicate AES clk is ready to be gated.
+              Verfy that the AES clk hint status within clkmgr reads 0 (AES is idle).
```

Comment:
Verfy -> Verify, and in line 747 and 750

Reply:
Done, thanks!